tests/unittests/graphTest.cpp

```diff
@@ -1202,3 +1202,28 @@ TEST(Graph, hookTest) {
   ASSERT_TRUE(ph);
   ASSERT_EQ(ph, in);
 }
+
+TEST(Graph, moduleSize) {
+  Module mod;
+
+  EXPECT_EQ(mod.dataSize(), 0);
+
+  mod.createConstant(ElemKind::FloatTy, {1}, "var");
+  EXPECT_EQ(mod.dataSize(), 4);
+
+  mod.createConstant(ElemKind::FloatTy, {1, 32, 32, 16}, "var2");
+  EXPECT_EQ(mod.dataSize(), 4 + 4 * 32 * 32 * 16);
```

Comment:
If you'd assign these constants to e.g. `const1` and `const2`, you could have a more future-proof check like:
`EXPECT_EQ(mod.dataSize(), const1->getType()->getSizeInBytes() + const2->getType()->getSizeInBytes());`

Reply:
I don't think testing getSizeInBytes() would be very useful because that is the mechanism used to return mod.dataSize() - you're essentially expecting X == X. A check like that would pass even if the implementation is wrong, whereas if we check against a constant we're verifying the implementation is behaving as expected.

Reply:
Sure that works for me.